benchexec/tools/concurrentwitness2test.py

```diff
@@ -26,7 +30,10 @@ def version(self, executable):
         return self._version_from_tool(executable)
 
     def cmdline(self, executable, options, task, rlimits):
-        return [executable, task.single_input_file] + options
+        witness_options = ["--witness"]
+        additional_options = get_witness_options(options, task, witness_options)
+        input_file = get_unique_non_witness_input_files(task)
```

Comment:
That is a very good idea!

Currently I used an enum as a compromise to have a single function but still be able to distinguish between the three options to access the files of the task. Hopefully that is appropriate for this case.

Reply:
> But why use the additional variable for the constant witness option and not just inline it? Then we at least have only two additional lines in many cases.

Since it was currently a list, I introduced a variable to make it easier to change, but you are right it makes little sense. I adjusted this with commit fcba870b6534a0dea0df438d3a16903fdfbdd756